build.sbt

```diff
@@ -360,11 +407,14 @@ lazy val spark = (project in file("connectors/spark"))
       case DepModuleInfo("ch.qos.logback", "logback-core", _) => true
       case DepModuleInfo("org.apache.xbean", "xbean-asm9-shaded", _) => true
       case DepModuleInfo("oro", "oro", _) => true
-    },
+      case DepModuleInfo("org.glassfish", "javax.json", _) => true
+      case DepModuleInfo("org.glassfish.hk2.external", "jakarta.inject", _) => true
+      case DepModuleInfo("org.antlr", "ST4", _) => true,
+    }
   )
 
 lazy val root = (project in file("."))
-  .aggregate(serverModels, client, server, cli, spark)
+  .aggregate(serverModels, client, server, cli, spark, serverAndClientShaded)
```

Comment:
is this needed? wont spark project automatically compile this project?

Reply:
I thought all modules have to be mentioned here. Why is serverModels here then ?

Reply:
Removed it, and everything's working fine.